### PR TITLE
Sample project COW

### PR DESCRIPTION
**Pull request by lamaralmp**
## Description
Sumamos el [proyecto de ejemplo de COW](http://github.com/mercadopago/checkout-payment-sample) a la docu. Con nota introdutória y card de GitHub

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

**Review by abderossi — APPROVED** [review]

File: guides/online-payments/checkout-pro/integration.es.md [hunk 1]
```diff
@@ -423,6 +426,14 @@ Por último, suma el siguiente código para mostrar el botón de pago de tu Chec
 #### ¡Excelente! Terminaste tu integración.
 _Haz clic en el link dentro de tu sitio y [prueba la integración de tu Checkout Pro](https://www.mercadopago.com.ar/developers/es/guides/online-payments/checkout-pro/test-integration/)_.<br/><br/>
 
+## Ejemplos descargables
+
+> GIT
+>
+> Checkout Pro
+>
+> Te dejamos <a href="http://github.com/mercadopago/checkout-payment-sample" target="_blank">ejemplos completos de integración</a> en GitHub para PHP o NodeJS para que puedas descargar al instante.
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/integration.es.md:435]:
```suggestion
> Te dejamos [ejemplos completos de integración](http://github.com/mercadopago/checkout-payment-sample) en GitHub para PHP o NodeJS para que puedas descargar al instante.
```

File: guides/online-payments/checkout-pro/integration.pt.md [hunk 2]
```diff
@@ -423,6 +426,14 @@ Por último, adicione o seguinte código para mostrar o botão de pagamento do s
 #### Excelente! Você concluiu sua integração.
 _Clique no link dentro do seu site e [teste a integração do seu Checkout Pro](https://www.mercadopago.com.br/developers/pt/guides/online-payments/checkout-pro/test-integration/)_.<br/><br/>
 
+## Exemplos para download
+
+> GIT
+>
+> Checkout Pro
+>
+> Disponibilizamos <a href="http://github.com/mercadopago/checkout-payment-sample" target="_blank">exemplos completos de integração</a> no GitHub para PHP ou NodeJS para que você possa fazer o download imediatamente.
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/integration.pt.md:435]:
```suggestion
> Disponibilizamos [exemplos completos de integração](http://github.com/mercadopago/checkout-payment-sample) no GitHub para PHP ou NodeJS para que você possa fazer o download imediatamente.
```

File: guides/online-payments/checkout-pro/integration.es.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # Integra Checkout Pro
 
+La integración del Checkout Pro de Mercado Pago te permite cobrar a través de nuestro formulario web desde cualquier dispositivo de manera simple, rápida y segura.
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/integration.es.md:3]:
```suggestion
Integrar el Checkout Pro de Mercado Pago te permite cobrar a través de nuestro formulario web desde cualquier dispositivo de manera simple, rápida y segura.
```

File: guides/online-payments/checkout-pro/integration.pt.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
 # Integre o Checkout Pro
 
+A integração por Checkout Pro do Mercado Pago te permite cobrar através do nosso formulário web de qualquer dispositivo de forma simples, rápida e segura.
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/integration.pt.md:3]:
```suggestion
Integrar Checkout Pro do Mercado Pago te permite cobrar através do nosso formulário web de qualquer dispositivo de forma simples, rápida e segura.
```

File: guides/online-payments/checkout-pro/integration.en.md [hunk 5]
```diff
@@ -1,5 +1,8 @@
 # Integrate Checkout Pro
 
+Integrating Mercado Pago's Checkout Pro allows you to charge your customers through our web form from any device in a simple, fast and secure way.
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/integration.en.md:3]:
```suggestion
Integrate Mercado Pago's Checkout Pro allows you to charge your customers through our web form from any device in a simple, fast and secure way.
```

**Review by pamalv7 — CHANGES_REQUESTED** [review]

**Review by abderossi — APPROVED** [review]

File: guides/online-payments/checkout-pro/integration.es.md [hunk 6]
```diff
@@ -1,5 +1,8 @@
 # Integra Checkout Pro
 
+Integrar el Checkout Pro de Mercado Pago te permite cobrar a través de nuestro formulario web desde cualquier dispositivo de manera simple, rápida y segura.
+Usa los [ejemplos descargables](#bookmark_ejemplos_descargables) para conocer la integración completa o para adaptarlos según lo que necesites.
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/integration.es.md:4]:
```suggestion

Usa los [ejemplos descargables](#bookmark_ejemplos_descargables) para conocer la integración completa o para adaptarlos según lo que necesites.
```

File: guides/online-payments/checkout-pro/integration.pt.md [hunk 7]
```diff
@@ -1,5 +1,8 @@
 # Integre o Checkout Pro
 
+Integrar Checkout Pro do Mercado Pago te permite cobrar através do nosso formulário web de qualquer dispositivo de forma simples, rápida e segura.
+Use os [exemplos para download](#bookmark_exemplos_para_download) para conhecer a integração completa ou para adaptá-los de acordo com o que precisa.
```
Comment (pamalv7) [inline guides/online-payments/checkout-pro/integration.pt.md:4]:
```suggestion

Use os [exemplos para download](#bookmark_exemplos_para_download) para conhecer a integração completa ou para adaptá-los de acordo com o que precisa.
```

**Review by pamalv7 — CHANGES_REQUESTED** [review]

**Review by abderossi — APPROVED** [review]
Estoy de acuerdo con el comentario de agregar un espacio que hizo @pamalv7

**Review by pamalv7 — APPROVED** [review]

**Pull request merged**